[docs][migrate] voyager reference section (#15654)
yugabyte/yugabyte-db: docs/content/preview/migrate/reference/_index.md

---
title: Reference
headerTitle: Reference
linkTitle: Reference
image: /images/section_icons/develop/learn.png
headcontent: Command line interfaces (CLIs), data modeling strategies and data type mapping reference.
type: indexpage
showRightNav: true
menu:
  preview:
    identifier: reference-voyager
    parent: voyager

> ---

Reviewer: known issues should prob go last in leftnav; or possibly under reference

Author: Will address this when I restructure known issues.

<div class="row">

  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
    <a class="section-link icon-offset" href="yb-voyager-cli/">
      <div class="head">
        <img class="icon" src="/images/section_icons/architecture/concepts.png" aria-hidden="true" />
        <div class="title">yb-voyager CLI</div>
      </div>
      <div class="body">

>         Learn about the CLI arguments and SSL connectivity for different phases of migration.

Reviewer: ```suggestion
        yb-voyager CLI commands, arguments, and SSL connectivity.
```

      </div>
    </a>
  </div>

  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
    <a class="section-link icon-offset" href="data-modeling/">
      <div class="head">
        <img class="icon" src="/images/section_icons/architecture/concepts/sharding.png" aria-hidden="true" />
        <div class="title">Data modeling</div>
      </div>
      <div class="body">

>         Learn about the sharding strategies supported and recommended for migration using YugabyteDB Voyager.

Reviewer: ```suggestion
        Learn about the sharding strategies to use when migrating using YugabyteDB Voyager.
```

      </div>
    </a>
  </div>

  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
    <a class="section-link icon-offset" href="datatype-mapping-mysql/">
      <div class="head">
        <img class="icon" src="/images/section_icons/reference/connectors/ecosystem-integrations.png" aria-hidden="true" />
        <div class="title">Datatype mapping</div>
      </div>
      <div class="body">

>        Learn about the list of supported data type mappings from different source databases to YugabyteDB.

Reviewer: ```suggestion
      Data type mapping from different source databases to YugabyteDB.
```
